src/rest_api/routing.clj

```diff
@@ -66,18 +66,19 @@
                        (apply merge (cons fields (->> this :widget vals)))
                        fields)
           route-data (assoc this :field field-defs)
-          entity-class (:entity-class route-data)]
+          entity-class (:entity-class route-data)
+          entity-segment (str/replace entity-class #"-" "_")]
```

Comment:
This is fine - just to note that this is why `rest-api.test-routing/test-defroutes` now needs to supply `entity-class` with a concrete string value in tests.